source/_docs/guides/getting-started/03-create-new-site.md

```diff
@@ -35,7 +35,7 @@ In this lesson, we’re going to create and configure a new WordPress or Drupal
 
 2. Select **<span class="glyphicons glyphicons-plus" aria-hidden="true"></span> Create New Site**. You’ll be asked to name this site and, if you’re part of an agency, associate this site with your organization.
 
-3. Select **Continue** and explore the available site types and pre-packaged distributions.
+3. Select **Continue** and explore the available site types and Custom Upstreams for public distributions.
```

Comment:
I second this. We can keep it simple for this level. This might be a place to socialize the term 'Upstream' but it may be too soon in the learning process to start adding insider terms. 'Site Types' while a bit awkward may be sufficient.

> Custom Upstreams for public distributions